docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  openwpm:
+    image: openwpm
+    environment:
+      # Pickup the AWS secret and access key from host env
```

Comment:
I don't think we should also bring in AWS credentials into the dockerfile. Some users may not want their credentials available to the OpenWPM container.